src/amo/components/Link/index.js

```diff
@@ -108,10 +112,24 @@ export class LinkBase extends React.Component {
     }
 
     return (
-      <Link {...customProps} to={linkTo}>
-        {children}
-        {needsExternalIcon ? <Icon name={iconName} /> : null}
-      </Link>
+      <Fragment>
```

Comment:
I don't think we need a `Fragment` here, you could keep the same code as before and spread a set of props like `customProps`:

```js
const linkProps = {
  ...customProps,
  target,
  rel: target === '_blank' ? 'noopener noreferrer' : undefined,
};

<Link {...linkProps} to={linkTo}>
  {children}
  {needsExternalIcon ? <Icon name={iconName} /> : null}
</Link>
```

Reply:
Good solution.

Reply:
@nabendu82 do you need further information to update your PR?